### PR TITLE
docs: Add "Defend Your Workstation" page

### LINKED ISSUE #43: Add a "Defend Your Workstation" page
## Overview

Add a new content overview page for the "Defend Your Workstation" topic/concept. It will be less of a guide and meant more as a landing page to drive folks to the appropriate local protection method. This is meant to answer the question of how Phylum can defend the developer by shifting security even further left.

## Additional context

Provide more detail around the new content:

* The page will be a new landing page
  * The content will be action oriented, with more detail and distinction made about the context (local development) within which those integrations exist
  * Provide a compelling user story to help sell the value offered by the integrations
* Cover all of the current options
  * Direct CLI use
  * "pre-check" default extensions
  * Git pre-commit hook
  * Optional CLI extensions
* Step through the process to make it as clear as possible
  * It should be possible for someone with no knowledge of Phylum to read through one of these pages and follow along to completion
* A single page is okay for now
  * Move to the "self-identify" (e.g., tile based decision tree) model later, with individual pages/docs for each combination
* Get more guidance from @furi0us333 and/or the marketing team

### PR DESCRIPTION
**Pull request by furi0us333**
Add "Defend Your Workstation" page to be used as a quick reference to get someone started with this feature.

Close #43

### CODE REVIEW

File: docs/home/defend_your_workstation.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+[phylum] Running build scripts inside sandbox…
+
+up to date in 122ms
+[phylum] Packages built successfully.
+```
+
+## Supported Ecosystems
+
+Phylum provides support for a number of common package managers used in the [supported ecosystems](https://docs.phylum.io/docs/analyzing-dependencies). These include `npm`, `yarn`, `pip`, `poetry`, `bundle`. `cargo` and more on the way! To get the current list, check the [official extensions](https://github.com/phylum-dev/cli/tree/main/extensions). The ecosystem extensions there are included by default when installing the Phylum CLI.
```
Comment (louislang) [inline docs/home/defend_your_workstation.md:99]: Currently `npm`, `yarn` and `pip` (and also maybe `poestry`?) leverage the sandbox. The `bundle` and `cargo` do not. idk if we want to call that out here or now.

Comment (furi0us333) [reply docs/home/defend_your_workstation.md:99]: If we can get the specifics, I think we can note that caveat.

Comment (maxrake) [inline docs/home/defend_your_workstation.md:99]:
```suggestion
Phylum provides support for a number of common package managers used in the [supported ecosystems](https://docs.phylum.io/docs/analyzing-dependencies). These include `npm`, `yarn`, `pip`, `poetry`, `bundle`, `cargo` and more on the way! To get the current list, check the [official extensions](https://github.com/phylum-dev/cli/tree/main/extensions). The ecosystem extensions there are included by default when installing the Phylum CLI.
```

File: docs/home/defend_your_workstation.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+Phylum uses a two-pronged approach to protect your system during package installation: package pre-check and an installation sandbox.
+
+## How It Works
+
+The Phylum CLI offers default ecosystem extensions that wrap package manager tools to protect commands which would install new packages or otherwise execute arbitrary code.
+
+When using Phylum to install a package with an ecosystem extension, the application will first query the Phylum platform for any findings related to the packages that would be installed. If the package findings fail the default policy, the installation is aborted.
+
+An additional layer of protection is provided with an installation sandbox. This ensures restricted resources like filesystem and network operations are enforced while executing arbitrary code. If any unwanted behavior is observed, Phylum will block the behavior and abort the installation before your system is impacted.
```
Comment (maxrake) [inline docs/home/defend_your_workstation.md:21]: The wording here is a bit vague on _when_ the sandbox is applied. Some readers may think it implies the sandbox is applied for _every_ ecosystem extension. Perhaps the wording can be better to call out that _some_ of the ecosystem extensions are protected with sandbox out of the box (as it were). These are currently the `npm`, `yarn`, and `pip` extensions.

**Review by maxrake — CHANGES_REQUESTED** [review]
One typo found and one question about sandbox language.

File: docs/home/defend_your_workstation.md (hunk 1 above)
Comment (maxrake) [reply docs/home/defend_your_workstation.md:99]: Dang...another instance of me leaving the review up too long and not seeing other comments before I made my own.

Comment (maxrake) [reply docs/home/defend_your_workstation.md:99]: `poetry` does not currently execute the package install with the sandbox.
Ref: https://github.com/phylum-dev/cli/blob/main/extensions/poetry/main.ts#L70-L73

Comment (furi0us333) [reply docs/home/defend_your_workstation.md]: fixed

File: docs/home/defend_your_workstation.md (hunk 2 above)
Comment (furi0us333) [reply docs/home/defend_your_workstation.md:21]: I added a note here

https://github.com/phylum-dev/documentation/blob/e807cfcd89cfcd794ce491b35c0997dc47b9fddc/docs/home/defend_your_workstation.md?plain=1#L101

File: docs/home/defend_your_workstation.md (hunk 1 above)
Comment (furi0us333) [reply docs/home/defend_your_workstation.md]: I added a note here

https://github.com/phylum-dev/documentation/blob/e807cfcd89cfcd794ce491b35c0997dc47b9fddc/docs/home/defend_your_workstation.md?plain=1#L101

File: docs/home/defend_your_workstation.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+up to date in 122ms
+[phylum] Packages built successfully.
+```
+
+## Supported Ecosystems
+
+Phylum provides support for a number of common package managers used in the [supported ecosystems](https://docs.phylum.io/docs/analyzing-dependencies). These include `npm`, `yarn`, `pip`, `poetry`, `bundle`, `cargo` and more on the way! To get the current list, check the [official extensions](https://github.com/phylum-dev/cli/tree/main/extensions). The ecosystem extensions there are included by default when installing the Phylum CLI.
+
+Note: The pre-check feature is available for all listed ecosystem extensions, but the installation sandbox component is currently only support for `npm`, `yarn`, and `pip`.
```
Comment (maxrake) [inline docs/home/defend_your_workstation.md:101]:
```suggestion
Note: The pre-check feature is available for all listed ecosystem extensions, but the installation sandbox component is currently only supported for `npm`, `yarn`, and `pip`.
```

**Review by maxrake — CHANGES_REQUESTED** [review]
One more small language change requested.

**Review by maxrake — APPROVED** [review]
LGTM

**Pull request merged**